fluentd/wait_for_es_version.rb

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env ruby
+
+require 'net/https'
+require 'json'
+
+if ARGV.length != 1
+  abort "Usage: wait_for_es_version.rb <min_es_version>"
+end
+
+min_version = ARGV[0]
+
+uri = URI('https://elasticsearch.openshift-logging.svc:9200')
```

Comment:
any reason we need to maybe pass this in as an arg?

Reply:
i was wondering this as well... it wouldn't be difficult to add it in. my initial thought was this was going to be specific for a CLO deployed fluentd/ES...